docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -154,10 +145,15 @@ NOTE: The cloud provider is preconfigured to the same value for both trial and p
 * *Availability zones*: Select `Multi`.
 ////
 
-. If you've purchased a subscription for {product-kafka}, use the slider to select an instance size. In {product-kafka}, you can create Kafka instances of 1 or 2 streaming units depending on the capacity required. If you're creating a trial instance, the instance size is preconfigured.
++
+
 . Click *Create instance* to start the creation process for your Kafka instance.
 +
 --
+[.screencapture]
+.Kafka production instance configuration details
+image::sak-configure-kafka-instance.png[Image of instance configuration details in Create Kafka instance window]
+
```

Comment:
Given that we've omitted this image from step 1, I don't think we actually need it anymore. I wouldn't put it in this step.

Reply:
@jbyrne-redhat Did we agree to leave this image out from the customer portal version of the guide? If so, I won't need to get an updated screenshot.

Reply:
@AedinC I haven't been in any conversations where we "agreed" to do this, per se. But my opinion is that we can do without it. It creates ongoing maintenance, and I don't think it adds a ton of value. 

An alternative would be to restore it to step 1 (as it is [currently](https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/f351c4bd-9840-42ef-bcf2-b0c9be4ee30a#_b4f95791-b992-429d-9e8e-cceb63ae829f)) and reword step 2 here  as "Define the following instance details" or "Define the instance details shown in the following list." (don't mention the name of the form, in other words).

I'm inclined towards option 1, so that we don't have to keep updating this figure every time the UI has a slight change.

Reply:
@jbyrne-redhat I do think we could add more images to our guides but this one adds no value that I can see so I'm going to remove it.